spec/unit/concerns/connection_spec.rb

```diff
@@ -60,6 +60,22 @@
         end
       end
     end
+
+    describe ":logger" do
+      let(:options) { { adapter: Faraday.default_adapter } }
+
+      before(:each) do
+        client.stub(:authentication_middleware)
+        client.stub(:cache)
+        client.stub(options: options)
+        Restforce.stub(log?: true)
+      end
+
+      it "must always be used last before the Faraday Adapter" do
+        client.middleware.handlers.pop(2).index(Restforce::Middleware::Logger).
```

Comment:
no like Array#pop? :) no prob. reverse is a nice trick here that didn't come to me.